include/grpc++/security/server_credentials.h

```diff
@@ -39,6 +39,7 @@
 
 #include <grpc++/security/auth_metadata_processor.h>
 #include <grpc++/support/config.h>
+#include <grpc/grpc_security.h>
```

Comment:
Yup. New header LGTM. Thanks!

On Tue, Apr 19, 2016 at 2:03 PM, deepaklukose <notifications@github.com>
wrote:

> In include/grpc++/security/server_credentials.h
> <https://github.com/grpc/grpc/pull/5958#discussion_r60310744>:
>
> > @@ -39,6 +39,7 @@
> >
> >  #include <grpc++/security/auth_metadata_processor.h>
> >  #include <grpc++/support/config.h>
> > +#include <grpc/grpc_security.h>
>
> Done. Can you take another look to ensure this is how you guys envisioned
> it.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc/pull/5958/files/dce598c9547dcb5340375232ce4dfdede369d048#r60310744>
>